packages/modules/widget/src/errors.ts

```diff
@@ -34,5 +71,6 @@ export class WidgetScriptModuleError extends Error {
         options?: ErrorOptions,
     ) {
         super(message, options);
+        this.name = WidgetScriptModuleError.toString();
```

Comment:
this wont work, when compiled the classname will be different

Reply:
will fix ;)